content/spin/v3/running-apps.md

```diff
@@ -155,6 +156,40 @@ For additional information about Spin's `watch` feature, please see the [Spin wa
 
 > The application manifests shown in the blog post are the version 1 manifest, but the content applies equally to the version 2 format. 
 
+## Splitting an Application Across Environments
+
+You can run a subset of the components in an application by using the `--component-id` (`-c`) flag. Set the flag multiple times to select multiple components.
+
+> This is an experimental feature. Unlike stable features, it may change even between minor versions.
+
+This supports _selective deployment_ - that is, splitting an application across environments while still distributing it as a single unit. For example, suppose your application contains the following layers:
+
+* A front end which handles browser and API requests, which you want to run in edge data centres close to users
+* A database backend which must be run in a single central location
+* An AI service which requires a GPU or other specialized (and costly) hardware
+
+You could deploy the application across the tiers by using `--component-id`. For example:
+
+<!-- @nocpy -->
+
+```console
+# on the US data centre edge nodes
+$ spin up -f registry.example.com/app:1.1.0 -c ui -c api -c assets
+
+# on the European data centre edge nodes
+$ spin up -f registry.example.com/app:1.1.0 -c ui -c api -c assets 
+
+# on the European data centre core nodes
+$ spin up -f registry.example.com/app:1.1.0 -c database
+
+# on the GPU hardware
+$ spin up -f registry.example.com/app:1.1.0 -c chat-engine -c sentiment-analyzer
+```
+
+> In practice you'd set these commands up in a scheduler or orchestrator rather than typing them interactively.
```

Comment:
SpinKube would use a SpinApp field rather than the `-c` option wouldn't it?  I was thinking here of environments where you would actually deal with the command form, e.g. Nomad jobs.  But will try to rephrase.

Reply:
Oh I see. I still think this is a good opportunity to reference SpinKube but leaving it as is makes sense too.